Source/WebCore/css/ComputedStyleExtractor.cpp

```diff
@@ -1842,7 +1842,7 @@ static CSSValueID valueIDForRaySize(RayPathOperation::Size size)
     return CSSValueInvalid;
 }
 
-static Ref<CSSValue> valueForPathOperation(const RenderStyle& style, const PathOperation* operation, SVGPathConversion conversion = SVGPathConversion::None)
+static RefPtr<CSSValue> valueForPathOperation(const RenderStyle& style, const PathOperation* operation, SVGPathConversion conversion = SVGPathConversion::None)
```

Comment:
This feels like sweeping the bug under the carpet. I don't see an explanation why this is ok.